plugins/colorschemes/ayu.nix

```diff
@@ -0,0 +1,48 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+} @ args:
+with lib; let
+  cfg = config.colorschemes.ayu;
+  helpers = import ../helpers.nix args;
+in {
+  options = {
+    colorschemes.ayu =
+      helpers.extraOptionsOptions
+      // {
+        enable = mkEnableOption "ayu";
+
+        package = helpers.mkPackageOption "ayu" pkgs.vimPlugins.neovim-ayu;
+
+        mirage = helpers.defaultNullOpts.mkBool false ''
+          Set to `true` to use `mirage` variant instead of `dark` for dark background.
+        '';
+
+        overrides =
```

Comment:
From the documentation:

Alternatively, overrides can be a function that returns a dictionary of the same format. You can use the function to override based on a dynamic condition, such as the value of 'background'.

Reply:
Done !